main/guides/getting-started/README.md

```diff
@@ -95,17 +117,26 @@ yarn build
 
 **Note:** MacOS, Linux, and WSL are currently supported. There is currently **no support** for native Windows 10 or native Windows 11.
 
+## Build the Cosmic Swingset Package
+
+```shell
+cd packages/cosmic-swingset
+make
```

Comment:
suggestion:

```suggestion
(cd packages/cosmic-swingset && make)
```

Doing the `cd` (and `make`) in a sub-shell means that the `cd ../..` below aren't necessary.

Reply:
Pretty slick. Change made.